app/Models/ReferenceSlide.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class ReferenceSlide extends Model
+{
+	protected $fillable = ['reference_color_id', 'reference_icon_id', 'slide_id', 'referencable_type', 'referencable_id'];
+	public $table = 'reference_slide';
+
+	/**
+	 * Get all of the owning referencable models.
+	 */
+	public function referencable()
+	{
+		return $this->morphTo();
+	}
+
+	public function slide()
+	{
+		return $this->belongsTo('App\\Models\\Slide');
```

Comment:
How about not using a string?
```suggestion
		return $this->belongsTo(App\Models\Slide::class);
```